front/src/applications/stdcmV2/components/StdcmSuggestions.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react';
+
+import { Input, type InputProps } from '@osrd-project/ui-core';
+import { isEmpty } from 'lodash';
+
+import SelectImprovedSNCF, {
+  type SelectOptionObject,
+} from 'common/BootstrapSNCF/SelectImprovedSNCF';
+
+export interface StdcmSuggestionsProps<T extends string | SelectOptionObject> extends InputProps {
+  options: T[];
+  onSelectSuggestion: (option: T) => void;
+}
+
+const StdcmSuggestions = <T extends string | SelectOptionObject>({
+  options,
+  onSelectSuggestion,
+  onFocus,
+  onBlur,
+  disabled,
+  ...rest
+}: StdcmSuggestionsProps<T>) => {
+  const [isSelectVisible, setIsSelectVisible] = useState(!isEmpty(options));
```

Comment:
By default, it should be close
```suggestion
  const [isSelectVisible, setIsSelectVisible] = useState(false);
```